rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -158,6 +158,26 @@ void getActiveFederationCreationBlockNumber_returnsGenesisFederationCreationBloc
             long activeFederationCreationBlockNumber = federationSupport.getActiveFederationCreationBlockNumber();
             assertThat(activeFederationCreationBlockNumber, is(genesisFederation.getCreationBlockNumber()));
         }
+
+        @Test
+        @Tag("getActiveFederatorBtcPublicKey")
+        void getActiveFederatorBtcPublicKey_returnsFederatorBtcPublicKeyFromGenesisFederation() {
+            byte[] activeFederatorBtcPublicKey = federationSupport.getActiveFederatorBtcPublicKey(0);
+            assertThat(activeFederatorBtcPublicKey, is(genesisFederation.getBtcPublicKeys().get(0).getPubKey()));
+        }
+
+        @Test
+        @Tag("getActiveFederatorBtcPublicKey")
+        void getActiveFederatorBtcPublicKey_withNegativeIndex_throwsIndexOutOfBoundsException() {
+            assertThrows(IndexOutOfBoundsException.class, () -> federationSupport.getActiveFederatorBtcPublicKey(-1));
+        }
+
+        @Test
+        @Tag("getActiveFederatorBtcPublicKey")
+        void getActiveFederatorBtcPublicKey_withIndexGreaterThanActiveFederationSize_throwsIndexOutOfBoundsException() {
+            int activeFederationSize = genesisFederation.getSize();
+            assertThrows(IndexOutOfBoundsException.class, () -> federationSupport.getActiveFederatorBtcPublicKey(activeFederationSize));
```

Comment:
not sure if these two tests should be here, since the check we are doing is not related to have two null federations.